modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -132,6 +132,10 @@ type HTTPTransport struct {
 	// to change or removal while experimental.
 	Versions []string `json:"versions,omitempty"`
 
+	// Specify the address to bind to when connecting to an upstream. In other words,
+	// it is the address the upstream sees as the remote address.
```

Comment:
Is that actually true, or is this just to help the local dialer choose the right interface to dial out of? Like could I set this to `1.1.1.1` or something ridiculous? :thinking:

Reply:
No, it doesn't accept any random address. An interface must exist with that address. The doc on `LocalAddr` field in [`net#Dialer`](https://pkg.go.dev/net#Dialer) says:

```
	// LocalAddr is the local address to use when dialing an
	// address. The address must be of a compatible type for the
	// network being dialed.
	// If nil, a local address is automatically chosen.
```

Reply:
Oh ok, perfect.

LGTM so far!